Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/securitycontextconstraints/user/mustrunas.go

```diff
@@ -51,16 +51,19 @@ func (s *mustRunAs) Validate(pod *api.Pod, container *api.Container) fielderrors
 	allErrs := fielderrors.ValidationErrorList{}
 
 	if container.SecurityContext == nil {
-		allErrs = append(allErrs, fmt.Errorf("Unable to validate nil security context for container %s", container.Name))
+		detail := fmt.Sprintf("unable to validate nil security context for container %s", container.Name)
+		allErrs = append(allErrs, fielderrors.NewFieldInvalid("securityContext", container.SecurityContext, detail))
 		return allErrs
 	}
 	if container.SecurityContext.RunAsUser == nil {
-		allErrs = append(allErrs, fmt.Errorf("Unable to validate nil RunAsUser for container %s", container.Name))
+		detail := fmt.Sprintf("unable to validate nil runAsUser for container %s", container.Name)
+		allErrs = append(allErrs, fielderrors.NewFieldInvalid("securitycontext.runAsUser", container.SecurityContext.RunAsUser, detail))
```

Comment:
Added to https://github.com/openshift/origin/issues/3092 unless anything else shakes out